apps/judicial-system/web/src/routes/Admin/Users/Users.tsx

```diff
@@ -84,7 +84,7 @@ export const Users: React.FC = () => {
   return (
     <div className={styles.userControlContainer}>
       <PageHeader title={formatMessage(titles.admin.users)} />
-      <div className={styles.logoContainer}>
+      <Box display="flex" marginBottom={9}>
```

Comment:
🙈 Curious George here, is Box > div in most cases? Is this just some clean up for consistency or is there a deeper reason?